src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx

```diff
@@ -58,8 +58,9 @@ Keep in mind that the default timeout for a new monitor is 180 seconds.
   >
     1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, then select **Synthetics**. Click <Icon name="fe-plus-circle"/> [**Create monitor**](https://one.newrelic.com/synthetics-nerdlets/monitor-create) in the far right of the Explorer nav.
     2. Specify a [monitor type](#setting-type), [name](#setting-name), and URL (may be any valid HTTP or HTTPS URL).
-    3. Optional: Add a **validation string** or **Advanced options**:
+    3. Optional: Add **device emulation**, a **validation string**, or **advanced options**:
 
+       * [Device emulation](/docs/synthetics/new-relic-synthetics/using-monitors/device-emulation) settings, including device type and screen orientation are available for simple browser monitors.
```

Comment:
@bpeckNR this link doesn't seem to work for device emulation. Should it be this: /docs/synthetics/synthetic-monitoring/using-monitors/device-emulation

Reply:
Great catch! I fixed this link.